cl/recap/management/commands/nightly_pacer_updates.py

```diff
@@ -242,4 +265,30 @@ def handle(self, *args, **options):
                 args=(d.pk, d.court_id), queue=queue
             )
 
+        for i, rd_id in enumerate(get_rd_ids_pray_and_pray()):
+
+            if i % 10 == 0:
+                logger.info(
+                    "Sent %s RECAP Documents to celery for crawling so far.", i
+                )
+
+            rd = (
+                RECAPDocument.objects.select_related("docket_entry__docket")
+                .only(
+                    "pacer_doc_id",
+                    "docket_entry_id",
+                    "docket_entry__docket_id",
+                    "docket_entry__docket__court_id",
+                    "is_sealed",
+                )
+                .get(pk=rd_id)
+            )
+
+            pacer_doc_id = rd.pacer_doc_id
+            if pacer_doc_id == "":
+                continue
+
+            else:
+                check_prayer_pacer.delay(rd.pk, None)
```

Comment:
This wont work. The current implementation of the `check_prayer_pacer` task needs a user ID. If the document isn't available, it calls the `prayer_unavailable` method to email the user.

Reply:
Perhaps we can use this PR to make the `user_id` optional and adjust the task's logic at the same time

Reply:
If it's a system user you need, we sometimes use the RECAP user for stuff like this.

Reply:
I think the `user_prayer` variable at line 395 will probably be `None`, which will cause a AttributeError exception .

Reply:
I think that's alright. This approach should work:

```diff

def prayer_unavailable(instance: RECAPDocument, user_pk: int) -> None:
    open_prayers = Prayer.objects.filter(
        recap_document=instance, status=Prayer.WAITING
    ).select_related("user")

    user_prayer = open_prayers.filter(user__pk=user_pk).first()
    
+   if not user_prayer: 
+       return

    email_recipients = [
        {
            "email": user_prayer.user.email,
            "date_created": user_prayer.date_created,
        }
    ]

-  if email_recipients:
   ...
```